### PR TITLE
Corrections for Choreo Connect documentation

### PR DESCRIPTION
**Pull request by slahirucd7**

### CODE REVIEW

**Review by praminda — APPROVED** [review]

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ API Manager product. Following is a list of key features supported by Choreo Con
 - Scope validation for JWT OAuth2 bearer tokens.
 - Applying CORS configuration from publisher portal.
 - Support full lifecycle states of APIs like created, prototyped, blocked, deprecated, published and etc.
-- Support virtual hosts(Vhosts) to expose the APIs.  
+- Support virtual hosts (Vhosts) to expose the APIs.  
 - API/Resource, application and subscription level throttling.
 - Advance throttling, custom policies and the blocking conditions.
 - Publish analytics to the Choreo cloud.
-- Back end JWT generation (Passing end user details to the back end services).
+- Back-end JWT generation (Passing end user details to the back end services).
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md:24]: backend is a one word.

Comment (slahirucd7) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md]: Corrected with: https://github.com/wso2/docs-apim/pull/4961/commits/ee1a0f528455b6e24a2ba1d2458f83fc4c2e740c

**Review by pubudu538 — APPROVED** [review]

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ APIM control plane, therefore the user actions like API deploying, application c
 
     - Download the latest [Choreo Connect release](https://github.com/wso2/product-microgateway/releases) and extract it to a folder of your choice. The extracted folder will be referred to as `CHOREO-CONNECT_HOME` here onwards.
 
-    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases) and follow the steps [here](https://github.com/wso2/product-apim#installation--running).
+    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases) and follow the steps described in [here](https://github.com/wso2/product-apim#installation--running).
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md:17]:
```suggestion
    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases) and follow the steps described [here](https://github.com/wso2/product-apim#installation--running).
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md [hunk 3]
```diff
@@ -26,9 +26,9 @@ Open the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml`
 
 In the `[controlPlane.eventHub]` section,
 
- - set `enabled` to true
- - update `serviceUrl` and `eventHub` endpoints with with the IP of API Manager. (Search for `apim` and replace them with the IP. Or else, add an entry to the `/etc/hosts` file as `<ip-of-apim> apim`)
- - if you want to use a Gateway Environment other than the default, update `environmentLabels` with the name of the new Gateway Environment. If not, leave the value `"Default"` as it is.
+ - Set `enabled` to true
+ - Update `serviceUrl` and `eventHub` endpoints with with the IP of API Manager. (Search for `apim` and replace them with the IP. Or else, add an entry to the `/etc/hosts` file as `<ip-of-apim> apim`)
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md:30]:
```suggestion
 - Update `serviceUrl` and `eventHub` endpoints with the IP of API Manager. (Search for `apim` and replace them with the IP. Alternatively, add an entry to the `/etc/hosts` file as `<ip-of-apim> apim`)
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md [hunk 4]
```diff
@@ -26,17 +26,17 @@ forward the request to the actual backend or reply back to the client with an er
 ### 3. Adapter
 
 Router and Enforcer have a data plane API that can be used to dynamically configure them.
-When an API needs to be deployed on the Choreo Connect we need to translate that API definition to the format understood by the Router(convert the API into routes) and Enforcer.
+When an API needs to be deployed on the Choreo Connect we need to translate that API definition to the format understood by the Router (convert the API into routes) and Enforcer.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md:29]:
```suggestion
When an API needs to be deployed in Choreo Connect, you need to translate that API definition to the format understood by the Router (convert the API into routes) and Enforcer.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ Extract the Choreo Connect distribution zip. Extracted folder will be called as
 ### Step 2 - Initialize an API project
 
 Navigate to a preferred workspace folder using the command line. This is the location that is used to store the Choreo Connect project.
-Run the following command to create a project named "petstore".  This will create the folder structure for the artifacts to be included. Use the -oas option to include the API definition to the project as follows.
+Run the following command to create a project named "petstore".  This will create the folder structure for the artifacts to be included. Use the --oas option to include the API definition to the project as follows.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-docker.md:25]:
```suggestion
Run the following command to create a project named "petstore". This creates the folder structure for the artifacts to be included. Use the --oas option to include the API definition to the project as follows.
```

**Pull request merged**